docs/AboutUs.adoc

```diff
@@ -11,11 +11,12 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 
 == Project Team
 
-=== John Doe
+=== Zeng Hou LIM 
 image::damithc.jpg[width="150", align="left"]
```

Comment:
I think you forgot to change your photo here 💃

Reply:
Ah, thanks for pointing it out! Will edit now @sunarjo-denny